src/viewmodels/motion.ts

```diff
@@ -1,3 +1,4 @@
+import { Console } from "console";
```

Comment:
```suggestion
import { Console } from "console";
```

Actually one quick alteration, can you remove this line

Reply:
Done